packages/fxa-auth-server/lib/routes/sms.js

```diff
@@ -1,147 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-'use strict';
-
-const error = require('../error');
-const isA = require('@hapi/joi');
-const PhoneNumberUtil = require('google-libphonenumber').PhoneNumberUtil;
-const validators = require('./validators');
-
-const METRICS_CONTEXT_SCHEMA = require('../metrics/context').schema;
-const FEATURES_SCHEMA = require('../features').schema;
-const TEMPLATE_NAMES = new Map([[1, 'installFirefox']]);
-
-module.exports = (log, db, config, customs, sms) => {
-  if (!config.sms.enabled) {
-    return [];
-  }
-
-  const REGIONS = new Set(config.sms.countryCodes);
-  const IS_STATUS_GEO_ENABLED = config.sms.isStatusGeoEnabled;
-
-  return [
-    {
-      method: 'POST',
```

Comment:
Just curious - when working on the Swagger API documentation, I noticed that there were a few routes that are noted as deprecated in the repo and current API doc (e.g., `/account/unlock/resend_code`). Would we want to note that the routes for sms and signinCodes are deprecated for the API docs too?

Reply:
@xlisachan Yea we can. I thought about returning an error allow those lines but figured it made more sense to rip it out instead lol.